src/views/Shipment.vue

```diff
@@ -99,6 +104,17 @@ export default defineComponent({
     IonThumbnail,
     IonTitle,
     IonToolbar,
+    Image
+  },
+  props: ["product"],
+  computed: {
+    progress() {
+      return 0;
```

Comment:
I think there should be a logic to generate this value

Reply:
Removed , as there isn't any need.